working/augmentation-libraries/feature-specification.md

```diff
@@ -1027,6 +1038,12 @@ language and our tools.
 
 ## Changelog
 
+## 1.17
+
+*   Introduce compile-time errors about wrong structures in the graph of
+    libraries and augmentation libraries formed by directives like `import`
+    and `import augment`.
```

Comment:
If there are issue numbers that this addresses, can you mention them here?

Reply:
Done.